templates/careers/company-culture/identity.html

```diff
@@ -0,0 +1,82 @@
+        <p>We also set out to create a productive work environment for those who want to contribute to the open source revolution we are creating - living and working wherever they prefer, regardless of their identity. We continue to seek ways to improve on these fronts.</p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule"/>
+      <div class="col">
+        <h1 class="p-heading--2">Impact on our hiring process</h1>
+      </div>
+      <div class="col">
+        <p>As a candidate to join the company, we will work hard to assess your relevant ability, minimising bias associated with identity.</p>
+        <p>Because we are open to hiring in every corner of the world, we must have a hiring process that does not depend too much on understanding local context. A resume is difficult to interpret without context - is that a good university trajectory, or a poor one? Was that a competitive placement, or not? We want to hire the best from every corner of the globe, and we cannot possibly have local context across the board.</p>
+        <p>In addition, underlying patterns of prejudice and privilege mean that people who come from the same place with different identities may not have had the same opportunity to reflect their ability in their resume. We should not amplify and reinforce local prejudice through our own hiring decisions.</p>
+        <p>For these reasons, we expect hiring leads to be more willing to move forward candidates from under-represented groups into the standard assessment stage, which is anonymous. This gives those candidates an opportunity to show what they can do on objective grounds, rather than relying on our own flawed ability to interpret a resume.</p>
+        <p>Many candidates who were previously rejected based on their resumes applying for a team, do very well in this assessment process and ultimately get strong positive feedback from that team during interviews.</p>
+      </div>
+    </div>
+</section>
+
+<section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule"/>
+      <div class="col">
+        <h1 class="p-heading--2">Proactive appointments</h1>
+      </div>
+      <div class="col">
+        <p>We must attract the best colleagues from around the world.</p>
+        <p>‘Best’ is always a complex calculation that weighs many factors. People are not mechanical - they have strengths and weaknesses, and we weigh those up in making hiring decisions. Do we hire a developer with more experience in the language we use? Or a better track record of documenting their work? Or more familiarity with open source and community? Or a better track record of learning quickly? We often have candidates who are the best in one of those elements but not all of them. The ‘best’ candidate is usually the one who will round out and strengthen the team.</p>
```

Comment:
People are not mechanical &mdash; they have...

It's not the standard dash

Reply:
you beat me to it! Didn't know you were taking a look, my bad :)